packages/client/_src/Resources/Blog/CreatePost.ts

```diff
@@ -0,0 +1,9 @@
+import { BlogPost, BlogPostId } from "@effect-ts-app/boilerplate-types/Blog"
+
+@allowAnonymous
+@allowRoles("user")
```

Comment:
Where can I find references about those decorators? Where do they come from?

Reply:
Click em in editor for source.
Their metadata is picked up during routing https://github.com/effect-ts-app/boilerplate/blob/main/apps/api-api/_src/lib/routing.ts#L25